libraries/AP_NavEKF3/AP_NavEKF3_core.h

```diff
@@ -367,7 +368,17 @@ class NavEKF3_core : public NavEKF_core_common
     // get timing statistics structure
     void getTimingStatistics(struct ekf_timing &timing);
 
+    // get solution data for the EKF-GSF emergency yaw estimator
+    // return false if data not available
+	bool getDataEKFGSF(float *yaw_composite, float *yaw_composite_variance, float yaw[N_MODELS_EKFGSF], float innov_VN[N_MODELS_EKFGSF], float innov_VE[N_MODELS_EKFGSF], float weight[N_MODELS_EKFGSF]);
+
+    // attempt to reset the yaw to the EKF-GSF value
+    // returns false if unsuccessful
+    bool EKFGSF_resetMainFilterYaw();
+
 private:
+    EKFGSF_yaw *yawEstimator = nullptr;
```

Comment:
should we have one per lane, or just run one yaw estimator total?

Reply:
Probably only need one. I have changed the default values of EK3_GSF_RUN and EK3_GSF_USE to 1